internal/deviations/deviations.go

```diff
@@ -475,6 +475,11 @@ func InterfaceRefConfigUnsupported(_ *ondatra.DUTDevice) bool {
 	return *interfaceRefConfigUnsupported
 }
 
+// SkipNtpNonDefaultVrf returns if we skip the ntp nondefault vrf case
+func SkipNtpNonDefaultVrf(_ *ondatra.DUTDevice) bool {
```

Comment:
As discussed offline please follow the new guidelines for adding these new deviations.

Here is the style guide - https://github.com/openconfig/featureprofiles/blob/main/internal/deviations/README.md
And here is a sample PR - https://github.com/openconfig/featureprofiles/pull/1663/files